<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PushForm</class>
 <widget class="QWidget" name="PushForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>41</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="cmdEdt">
     <property name="text">
      <string>Редакт.</string>
     </property>
     <property name="icon">
      <iconset theme="accessories-text-editor"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cmdNew">
     <property name="text">
      <string>Нов.</string>
     </property>
     <property name="icon">
      <iconset theme="document-new"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cmdDel">
     <property name="text">
      <string>Удал.</string>
     </property>
     <property name="icon">
      <iconset theme="edit-delete"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cmdWrite">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Запись</string>
     </property>
     <property name="icon">
      <iconset theme="document-save"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cmdEsc">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сброс</string>
     </property>
     <property name="icon">
      <iconset theme="edit-undo"/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
